testapp/testapp/settings/base.py

```diff
@@ -165,3 +165,7 @@
 
 # Default field for primary keys
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+
+# Default heading level for care cards
+# Uncomment the line below and change the value to what you want (min=2 max=6)
+# DEFAULT_CARE_CARD_HEADING_LEVEL = "3"
```

Comment:
This should be `DEFAULT_CARE_CARD_HEADING_LEVEL = 3` without the quotes probably?

Reply:
I believe it has been converted to a str @mikemonteith

Reply:
It does get converted to a string but only for the help text.

Reply:
Removed the quotes.